engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -198,6 +203,22 @@ public HibernateValidatorConfiguration clockProvider(ClockProvider clockProvider
 	}
 
 	@Override
+	public HibernateValidatorConfiguration addValueExtractor(ValueExtractor<?> extractor) {
+		Contracts.assertNotNull( extractor, MESSAGES.parameterMustNotBeNull( "extractor" ) );
+
+		// TODO prevent duplicates
```

Comment:
Let's decide what to do about this before merging and implement it. We have added a lot of TODOs lately.